content/en/data_streams/_index.md

```diff
@@ -41,19 +43,15 @@ Data Streams Monitoring provides a standardized method for teams to understand a
 
 ## Setup
 
-To get started, follow the installation instructions to configure services with Data Streams Monitoring:
+For installation instructions and lists of supported technologies, choose your language:
 
 {{< partial name="data_streams/setup-languages.html" >}}
 
 <br/>
 
-| Runtime | Supported technologies |
-|---|----|
-| Java/Scala | Kafka (self-hosted, Amazon MSK, Confluent Cloud / Platform), RabbitMQ, HTTP, gRPC, Amazon SQS, Amazon Kinesis, Amazon SNS, Google Pub/Sub |
-| Python | Kafka (self-hosted, Amazon MSK, Confluent Cloud / Platform), RabbitMQ, Amazon SQS, Amazon Kinesis, Amazon SNS |
-| .NET | Kafka (self-hosted, Amazon MSK, Confluent Cloud / Platform), RabbitMQ, Amazon SQS, Amazon SNS, IBM MQ, Azure Service Bus |
-| Node.js | Kafka (self-hosted, Amazon MSK, Confluent Cloud / Platform), RabbitMQ, Amazon SQS, Amazon Kinesis, Amazon SNS, Google Pub/Sub |
-| Go | All (with [manual instrumentation][1]) |
+### Compatibility
+
+Data Streams Monitoring instruments Kafka _clients_ (consumers/producers), not Kafka brokers. If you can instrument your client infrastructure, you can use Data Streams Monitoring.
```

Comment:
I would put that in the installation step, not in `Compatibility`.
We give information about Kafka brokers from the point of view of clients. From this section, customer might think we don't give them information about Kafka brokers 🤔